packages/nodes-base/nodes/Clockify/ClockifyWriter.node.ts

```diff
@@ -0,0 +1,529 @@
+		},
+		inputs: ['main'],
+		outputs: ['main'],
+		credentials: [
+			{
+				name: 'clockifyApi',
+				required: true,
+			}
+		],
+		properties: [
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				options: [
+					{
+						name: 'Create',
+						value: 'create'
+					},
+					{
+						name: 'Update',
+						value: 'update'
+					},
+					{
+						name: 'Delete',
+						value: 'delete'
+					},
+				],
+				default: 'create',
+				description: 'The operation you wish to carry out.',
+			},
+			{
+				displayName: 'Resource',
+				name: 'resource',
+				type: 'options',
+				options: [
+					{
+						name: 'Project',
+						value: 'project'
+					},
+					{
+						name: 'Tag',
+						value: 'tag'
+					},
+					{
+						name: 'Time Entry',
+						value: 'timeEntry'
+					},
+				],
+				default: 'project',
+				description: 'The resource to operate on.',
+			},
+			{
+				displayName: 'Workspace',
+				name: 'workspaceId',
+				type: 'options',
+				typeOptions: {
+					loadOptionsMethod: 'listWorkspaces',
+				},
+				required: true,
+				default: [],
+			},
+			{
+				displayName: 'User',
+				name: 'userId',
+				type: 'options',
+				typeOptions: {
+					loadOptionsDependsOn: ['workspaceId'],
+					loadOptionsMethod: 'loadUsersForWorkspace',
+				},
+				required: true,
+				default: [],
+				displayOptions: {
+					show: {
+						resource: [
+							'timeEntry',
+						],
+					},
+				},
+			},
+			{
+				displayName: 'Client',
+				name: 'clientId',
+				type: 'options',
+				typeOptions: {
+					loadOptionsDependsOn: ['workspaceId'],
+					loadOptionsMethod: 'loadClientsForWorkspace',
+				},
+				required: true,
+				default: [],
+			},
+			{
+				displayName: 'Project',
+				name: 'project',
+				type: 'options',
+				typeOptions: {
+					loadOptionsDependsOn: ['workspaceId'],
+					loadOptionsMethod: 'loadProjectsForWorkspace',
+				},
+				required: true,
+				default: [],
+				description: 'Project to associate with, leaving blank will use the project associated with the task.',
+				displayOptions: {
+					hide: {
+						resource: [
+							'project',
+						],
+					},
+				}
+			},
+			{
+				displayName: 'Billable?',
+				name: 'billable',
+				type: 'boolean',
+				required: true,
+				default: false,
+				displayOptions: {
+					hide: {
+						resource: [
+							'tag',
+						],
+					},
+				}
+			},
+			//Project Properties
+			{
+				displayName: 'Project Name',
+				name: 'projectName',
+				type: 'string',
+				required: true,
+				default: '',
+				description: 'Name of project being created.',
+				displayOptions: {
+					show: {
+						resource: [
+							'project',
+						],
+					},
+				}
+			},
+			{
+				displayName: 'Project Color',
+				name: 'color',
+				type: "color",
+				required: true,
+				default: '#0000FF',
+				displayOptions: {
+					show: {
+						resource: [
+							'project',
+						],
+					},
+				}
+			},
+			{
+				displayName: 'Public?',
```

Comment:
This also has double-punctuation.
```suggestion
				displayName: 'Public',
```